.github/workflows/linter.yml

```
@@ -31,3 +31,5 @@ jobs:
           VALIDATE_GOOGLE_JAVA_FORMAT: false
           # Too sensitive.
           VALIDATE_JSCPD: false
+          # No android in here
+          VALIDATE_KOTLIN_ANDROID: false
```

I think we can simply disable the check for the files where the linting fails.

```suggestion
          VALIDATE_KOTLIN_ANDROID: false
          # Ignore files that are failed to be linted without error message
          FILTER_REGEX_EXCLUDE: <all broken sh files>
```